docs/4.2/gcp_guide.md

```diff
@@ -0,0 +1,203 @@
+# Running Teleport on GCP
+
+We've created this guide to give customers a high level overview of how to use Teleport
+on [Google Cloud](https://cloud.google.com/gcp/) (GCP). This guide provides a high level introduction leading to
+a deep dive into how to setup and run Teleport in production.
+
+We have split this guide into:
+
+- [Teleport on GCP FAQ](#teleport-on-gcp-faq)
+- [GCP Teleport Introduction](#gcp-teleport-introduction)
+- [GCP Quickstart](#gcp-quickstart)
+
+## Teleport on GCP FAQ
+#### Why would you want to use Teleport with GCP?
+As leader in [Beyond Corp](https://cloud.google.com/beyondcorp/), GCP already provides
+some great tools out of the box such as [Cloud Identity - Aware Proxy](https://cloud.google.com/iap/).  
```

Comment:
`Identity - Aware` -> `Identity-Aware` (extra spaces, again Google's way of formatting it)